app/views/collections/_edit_slides_in_collection.html.erb

```diff
@@ -1,8 +1,6 @@
-<div class="panel panel-primary">
-  <div class="panel-heading">
-    <h3 class="panel-title">Edit slides in collection</h3>
-  </div>
-  <div class="panel-body">
+<h1>Edit slides in collection</h1>
```

Comment:
If I'm following this correctly, this goes under the `Editing Collection` H1. There should only be one H1 per page and it should usually be similar to the `<title>` element.
Set this and the main `Edit collection` section to `h2`